homeassistant/util/package.py

```diff
@@ -28,8 +30,13 @@ def is_virtual_env() -> bool:
 
 @cache
 def is_docker_env() -> bool:
-    """Return True if we run in a docker env."""
-    return Path("/.dockerenv").exists()
+    """Return True if we run in a container env."""
+    return (
+        Path("/run/.containerenv").exists()
+        or Path("/.dockerenv").exists()
```

Comment:
We should do the one that exists in our supported installation first.

```suggestion
        Path("/.dockerenv").exists().exists()
        or Path("/run/.containerenv")
```